### DISCUSSION TITLE
알고리즘 포스팅

### DISCUSSION THREAD

**Discussion by saejinpark** [DeveloperAcademy-POSTECH/.github#60] [Idea]
안녕하세요 3기 러너 제패토입니다
최근에는 잘못했지만 Velog에 소소하게 알고리즘 문제풀이를 하고있습니다

이전에 풀었던 좌표문제 풀이에요

https://velog.io/@a590824/2024%EB%85%84-7%EC%9B%94-11%EC%9D%BC-%EB%AC%B8%EC%A0%9C%ED%92%80%EC%9D%B4

**Comment by yeniful** [comment]
안녕하세요 제페토! 어떤 알고리즘 포스팅인지 제목과 내용에서 알려주시면 더 좋을 것 같아요 : )
